src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -44,9 +44,9 @@ def get_cookies(self):
 
     @keyword
     def get_cookie_value(self, name):
-        """Returns value of cookie found with `name`.
+        """Returns value of cookie found with ``name``.
 
-        If no cookie is found with `name`, this keyword fails.
+        If no cookie is found with ``name``, this keyword fails.
```

Comment:
I guess we could silently deprecate this keyword in favor of `Get Cookie Information`.

Reply:
Good idea, will do.